cypress.json

```diff
@@ -1,5 +1,6 @@
 {
   "projectId": "odida5",
   "baseUrl": "http://localhost:9001",
-  "supportFile": "cypress/support/index.ts"
+  "supportFile": "cypress/support/index.ts",
+  "watchForFileChanges": false
```

Comment:
I added this. I think it's annoying to write a bit of test code, then go back to Cypress to find out what I'm querying the DOM for and the Cypress test starts over again increasing time to write tests rather than decreasing it.